firefox-ios/Client/Frontend/TrackingProtection/TrackigProtectionRedux/TrackingProtectionState.swift

```diff
@@ -134,11 +201,13 @@ struct TrackingProtectionState: StateType, Equatable, ScreenState {
                 trackingProtectionEnabled: state.trackingProtectionEnabled,
                 connectionSecure: state.connectionSecure,
                 showDetails: false,
-                showBlockedTrackers: false
+                showBlockedTrackers: false,
+                showsClearCookiesAlert: false,
+                shouldClearCookies: false,
+                shouldUpdateBlockedTrackerStats: false
             )
         default:
             return state
```

Comment:
we should be returning a new instance for our default, we discussed something during our team meeting, probably should have it documented somewhere. `return TrackingProtectionState(...)`